tests/ClientWrapperTest.php

```diff
@@ -135,4 +136,55 @@ public function testClientsAreProperlyConfigured(): void
         self::assertSame($backendConfiguration, $branchClient->getBackendConfiguration());
         self::assertSame($branchClient, $clientWrapper->getBranchClient());
     }
+
+    /**
+     * @dataProvider useBranchStorageDataProvider
+     */
+    public function testCreateBranchStorage(?bool $useBranchStorage, string $expectedClassName): void
```

Comment:
Ani s jednim parametrem tu nepracujes :)

Reply:
😆  Tak to je skvělej test, hlavně že to pokrývá všechny možnosti...
fixed